### ISSUE TITLE
The graph has low accessibility (dark blue dots on a black background)

### ISSUE THREAD

**Issue by tedsecretsource** [gabrielchua/RAGxplorer#9]
**Describe the bug**
The contrast ratio of the graph is somewhat low contributing to low accessibility.

**To Reproduce**
1. Upload a file
2. Enter search terms and press Enter
3. The graph displays on a black background with dark blue dots representing the vectors

**Expected behavior**
The contrast ratio between the dark blue dots and the background is great enough to more easily see the distribution.

**Comment by gabrielchua** [comment]
good point! 

Two ways:
1) only have a “light” mode (there’s both in streamlit)
2) change the colours

Let me try 2)

**Comment by gabrielchua** [comment]
Changed the colors in #11 

Hope they're better/more accessible! 

<img width="1355" alt="Screenshot 2024-01-25 at 12 50 57 AM" src="https://github.com/gabrielchua/RAGxplorer/assets/34762009/f4fea992-1007-42fd-a9ea-b3fc5e5a3997">


<img width="1355" alt="Screenshot 2024-01-25 at 12 51 50 AM" src="https://github.com/gabrielchua/RAGxplorer/assets/34762009/cb674657-85f2-473e-914f-98c90b56dc97">
